en/scripting/subpackage.md

```diff
@@ -42,8 +44,8 @@ When the **sub-package** is completed, a callback is triggered, and if the loadi
 ```javascript
 cc.loader.downloader.loadSubpackage('01_graphics', function (err) {
     If (err) {
```

Comment:
The code: **If** -> **if**

Reply:
Sorry. Should not have made this mistake.